apps/public/cypress/integration/navigation.ts

```diff
@@ -40,7 +40,7 @@ describe("Navigating around the site", function() {
     cy.contains("Endorsement Disclaimers")
 
     // Click on the listings page link in the header nav
-    cy.get("header a")
+    cy.get(".navbar")
```

Comment:
Looks like naming of this section changed and tests were broken on master, so I updated "header" to "navbar"